RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py

```diff
@@ -89,6 +91,7 @@
     'keep CTPPSPixelDigiedmDetSetVector_ctppsPixelDigis_*_*',
     'keep CTPPSPixelClusteredmDetSetVector_ctppsPixelClusters_*_*',
     'keep CTPPSPixelRecHitedmDetSetVector_ctppsPixelRecHits_*_*',
+    'keep CTPPSPixelLocalTrackedmDetSetVector_ctppsPixelLocalTracks_*_*',
```

Comment:
In retrospect, it is perhaps much more economical to defined AOD first, then derive RECO from it, then derive FEVT from that.
Something to consider for a future cleanup.